Makefile

```diff
@@ -5,7 +5,7 @@ PREFIX ?= nginx-kubernetes-gateway
 GIT_COMMIT = $(shell git rev-parse HEAD)
 DATE = $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 
-TARGET ?= local
+TARGET ?= edge
```

Comment:
did you mean to update `VERSION` instead of `TARGET`?

Reply:
ahh yes. Thanks for catching!